cfgov/jinja2/v1/_includes/molecules/notification.html

```diff
@@ -0,0 +1,55 @@
+{# ==========================================================================
+
+   notification.render()
+
+   ==========================================================================
+
+   Description:
+
+   Builds notification markup when given:
+
+   options (optional):  An object used to customize the notification markup.
+
+   options.type:        The type of notification to display.
+                        Values may be: 'success', 'error', or 'warning'.
+
+   options.message:     Text to display within the notification markup.
+                        Default is 'Success.', 'Warning!', or 'Error!',
+                        depending on notification type.
+
+   options.explanation: Text to display as an explanation
+                        within the notification markup.
+
+   ========================================================================== #}
+{% macro render(options={}) -%}
+    {% set type  =  options.type if options.type in ['success', 'warning', 'error'] else 'success' %}
+    {% set icon_lookup  = {
+          'success' : 'approved',
+          'warning' : 'error',
+          'error'   : 'delete'
+        }
+    %}
+
+    {% if type == 'success' %}
+        {% set message  =  options.message | default('Success.') %}
+    {% elif type == 'warning' %}
+        {% set message  =  options.message | default('Warning!') %}
+    {% elif type == 'error' %}
+        {% set message  =  options.message | default('Error!') %}
+    {% endif %}
+
+    {% set notification_type  =  ('m-notification__' + type) if type %}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/6e40079dacf78a192e5d1d74371155c22fb54e9e